CMakeLists.txt

```diff
@@ -730,6 +730,17 @@ option(SWIFT_ENABLE_BACKTRACING
 set(SWIFT_DARWIN_DEPLOYMENT_VERSION_MACCATALYST "14.5" CACHE STRING
   "Minimum deployment target version for macCatalyst")
 
+# A tempoarary hack: force enabling HOSTTOOLS mode on Windows.
+# Right now, SwiftCompilerSources cannot be enabled for lldb because the
+# Windows build script builds lldb before swift and that fails with a missing
+# dependency on swiftrt (which is built within swift).
```

Comment:
lldb is not built before swift, it is built alongside as a single unified build. This implies that there is a missing dependency in the LLDB build definition.

Reply:
Thanks for clarifying. I updated the comment (We can fix the underlying problem in a follow-up PR).